### DISCUSSION TITLE
How to access a specific pinia store in the boot?

### DISCUSSION THREAD

**Discussion by realtebo** [quasarframework/quasar#13961] [General - Components / Directives / etc]
I've 2 pinia store (a pinia store with 2 modules ? i don't know what is the right termin sorry)

I would like to do things in the boot to initialize the state of a specific store/module

Could you help me please?

**Comment by MilosPaunovic** [comment]
You have your store as a parameter of `boot` function:

[https://quasar.dev/quasar-cli-webpack/boot-files#anatomy-of-a-boot-file](https://quasar.dev/quasar-cli-webpack/boot-files#anatomy-of-a-boot-file)

```js
export default ({ app, router, store }) => {
  // something to do
}
```

**Comment by realtebo** [comment]
This store is _the full pinia_ store. is not my specific one .

I discovered I can do 

`const my_specific_router = useMySpecifiStore(store);`

**Comment by alexrififi** [comment]
Don't bother, just use it like this

```ts
// boot/file.ts
import { boot } from 'quasar/wrappers'
import { useCategories } from '~/stores'

export default boot(async () => {
  useCategories().fetch()
})
```
